src/converters/app-converter.ts

```diff
@@ -235,7 +296,16 @@ export class StackConverter extends ArtifactConverter {
         const dependsOn = getDependsOn(resource);
         return {
             parent: parent,
-            dependsOn: dependsOn !== undefined ? dependsOn.map((id) => this.resources.get(id)!.resource) : undefined,
+            dependsOn:
+                dependsOn !== undefined
+                    ? dependsOn.flatMap((id) => {
+                          const resource = this.resources.get(id);
+                          if (resource?.otherResources && resource.otherResources.length > 0) {
+                              return resource.otherResources;
+                          }
+                          return resource!.resource;
```

Comment:
I know we didn't do it before either, but should we handle `resource` being unset and throw an error? Or is this guaranteed to be set?